target/linux/ar71xx/config-4.4

```diff
@@ -113,6 +113,7 @@ CONFIG_ATH79_MACH_GL_INET=y
 CONFIG_ATH79_MACH_GL_MIFI=y
 CONFIG_ATH79_MACH_GS_MINIBOX_V1=y
 CONFIG_ATH79_MACH_GS_OOLITE=y
+CONFIG_ATH79_MACH_HIVEAP_121=y
```

Comment:
@pepe2k Is this new for the ar71xx target? I ask as this does not seem to be standard atm for the rest of the NAND targets in ar71xx. (Ex, Meraki MR18, Airtight C-60, etc).